assets/sass/custom.sass

```diff
@@ -984,6 +980,10 @@ $os-selector-mobile-height: 48px
     background-image: url('/img/logos/partnerLogos/rit.svg')
     width: 96px
 
+  .epicgames-logo
+    background-image: url('/img/logos/partnerLogos/epicgames.svg')
+    width: 120px
```

Comment:
nit: these were originally in alphabetical order too, can you fix them? Thanks :D